### PR TITLE
CERNBox etherpad proposal

### PR DESCRIPTION
**Pull request by gmgigi96**

### CODE REVIEW

File: _gsocproposals/2022/proposal_CERNBox-Etherpad.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Etherpad plugin as a ScienceMeshDocs editor
+layout: gsoc_proposal
+project: CERNBox
+year: 2022
+organization:
```
Comment (aflp91) [inline _gsocproposals/2022/proposal_CERNBox-Etherpad.md:6]: Project difficulty, duration and mentor availability required:
    difficulty: [medium or high]
    duration: [350 or 175 hours]
    mentor_avail: [month interval]

File: _gsocproposals/2022/proposal_CERNBox-Etherpad.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Task ideas
+
+* Analyse the Etherpad plugins system and APIs, and identify the operations required to fetch/push documents 
+* Implement the connector in Etherpad and in the CS3 WOPI server
+* Test the integration, possibly writing appropriate unit tests for the WOPI server
+* Document the Etherpad connector and contribute it upstream
+
+## Expected results
+By the end of GSoC 2022 we expect an Etherpad plugin that can connect to CS3-compatible sync&syare systems.
```
Comment (aflp91) [inline _gsocproposals/2022/proposal_CERNBox-Etherpad.md:30]: `sync&syare` => `sync&share`

File: gsoc/2022/mentors.md [hunk 3]
```diff
@@ -27,3 +27,6 @@ layout: plain
 * Mayank Sharma[Mayank Sharma](mailto:mayank.sharma@cern.ch) CERN
 * Mark Smith [mark.smith1@imperial.ac.uk](mailto:mark.smith1@imperial.ac.uk) imperialcollege
 * Anna Zaborowska[Anna Zaborowska](mailto:anna.zaborowska@cern.ch) CERN
+* Giuseppe Lo Presti[giuseppe.lopresti@cern.ch](mailto:giuseppe.lopresti@cern.ch) CERN
```
Comment (aflp91) [inline gsoc/2022/mentors.md:30]: Please, add the mentors in alphabetical order.

**Review by aflp91 — CHANGES_REQUESTED** [review]
See my comments attached to files

File: _gsocproposals/2022/proposal_CERNBox-Etherpad.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  - CERN
+  - WWU
+  - ScienceMesh
+---
+
+## Description
+
+[CERNBox](https://cernbox.web.cern.ch) is a sync and share, collaborative cloud storage solution built at CERN on top of EOS. The service is used by more than 37K users and stores over 15PB of data. CERNBox has responded to the high demand in our diverse community to an easily and accessible cloud storage solution, which provides integrations with other CERN services for Big Science: visualisation tools, interactive data analysis and real-time collaborative editing. For the latter, a number of applications have already been integrated, by leveraging the [WOPI](https://docs.microsoft.com/en-us/microsoft-365/cloud-storage-partner-program/rest) open specifications.
+
+[ScienceMesh](https://cs3mesc4eosc.eu) is an emerging pan-European federated cloud infrastructure, which aims at bringing together several sync and share platforms across Europe. In the context of ScienceMesh the integration of CodiMD has been demonstrated through WOPI _bridge_ extensions.
```
Comment (hegner) [inline _gsocproposals/2022/proposal_CERNBox-Etherpad.md:16]: I believe this is not the link you want to have...

**Review by hegner — CHANGES_REQUESTED** [review]

File: _gsocproposals/2022/proposal_CERNBox-Etherpad.md (hunk 4 above)
Comment (glpatcern) [reply _gsocproposals/2022/proposal_CERNBox-Etherpad.md:16]: Thanks Benedikt and good point, we now have https://sciencemesh.io that is more appropriate than the institutional EU project. Will also fix the other comments.

**Review by aflp91 — APPROVED** [review]
Looks fine

**Pull request merged**